src/client/src/ui/blotter/views/blotter.scss

```diff
@@ -5,7 +5,9 @@
   width: 100%;
   overflow: hidden;
   font-family: 'Segoe UI', sans-serif;
-
+  .fixedDataTableCellLayout_wrap1 {
```

Comment:
Not quite sure about the naming of this, ie. ending with "wrap1". What's the reason for this?